CIP-0043/README.md

```diff
@@ -78,51 +79,62 @@ functions, these are:
+themselves. This creates different expecations for each argument for both of
+these schemes; we describe these below.
 
-For the ECDSA signature scheme, we have the following requirements:
+For the [ECDSA signature
+scheme](https://en.bitcoin.it/wiki/Elliptic_Curve_Digital_Signature_Algorithm),
+the requirements are as follows. Note that these differ from the serialization
+used by Bitcoin.
 
+* The verification key must correspond to the _(x, y)_ coordinates of a point 
+  on the SECP256k1 curve, where _x, y_ are unsigned integers in big-endian form.
 * The verification key must correspond to a result produced by
   [``secp256k1_ec_pubkey_serialize``](https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L394), 
   when given a length argument of 33, and the ``SECP256K1_EC_COMPRESSED`` flag.
   This implies all of the following:
     * The verification key is 33 bytes long.
-    * The first byte corresponds to the sign of the _y_ coordinate; this is
+    * The first byte corresponds to the parity of the _y_ coordinate; this is
       `0x02` if _y_ is even, and `0x03` otherwise.
-    * The remaining 32 bytes are the bytes of the _x_ coordinate, as a
-      big-endian integer.
+    * The remaining 32 bytes are the bytes of the _x_ coordinate.
 * The input to verify must be a 32-byte hash of the message to be checked. We 
   assume that the caller of `verifyEcdsaSecp256k1Signature` receives the 
   message and hashes it, rather than accepting a hash directly: doing so 
   [can be dangerous](https://bitcoin.stackexchange.com/a/81116/35586).
   Typically, the hashing function used would be SHA256; however, this is not
   required, as only the length is checked.
+* The signature must correspond to two unsigned integers in big-endian form;
+  henceforth _r_ and _s_.
 * The signature must correspond to a result produced by
   [``secp256k1_ecdsa_serialize_compact``](https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L487).
   This implies all of the following:
     * The signature is 64 bytes long.
-    * The first 32 bytes are the bytes of `r`, as a big-endian integer.
-    * The last 32 bytes are the bytes of `s`, as a big-endian integer.
+    * The first 32 bytes are the bytes of _r_.
+    * The last 32 bytes are the bytes of _s_.
 
-For the Schnorr signature scheme, we have the following requirements:
+For the Schnorr signature scheme, we have the following requirements, as
+described in the requirements for [BIP-340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki):
 
-* The verification key must follow the
-  [BIP-340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki)
-  standard for encoding, as emboded by the [``secp256k1_xonly_pubkey_serialize``](https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1_extrakeys.h#L61).
+* The verification key must correspond to the _(x, y)_ coordinates of a point 
```

Comment:
The same as above. Probably similar phrasing to that of the ECDSA verification keys would be good.